src/datashapes/nft-list.datashape.ts

```diff
@@ -0,0 +1,7 @@
+import { Meta } from '../entities/wallet-nft.entity';
+
+export const TYPE = 'nft-list';
+
+export type NFTListDatashape = {
```

Comment:
I am still suspicious of the narrow data shapes, but I am ready to experiment here :)